docs/user-guide/projected-volume/index.md

```diff
@@ -0,0 +1,72 @@
+---
+assignees:
+- jpeeler
+- pmorie
+title: Using Projected volumes
+---
+
+The projected volume is a volume that projects several existing volume sources
+into the same directory. Currently the supported projections are configmaps,
+downward API, and secrets. The resulting pod spec is also shorter when
+projecting to a single volume as opposed to multiple different locations. See
+[all-in-one volume design document](https://github.com/kubernetes/community/blob/{{page.githubbranch}}/contributors/design-proposals/all-in-one-volume.md)
+for more information.
+
+* TOC
+{:toc}
+
+## Overview of a projected volume
+
+The projected volume encapsulates multiple volumes to be projected, with each
+volume source respecting nearly the same parameters as supported by each
+individual type. Consider the following example:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: volume-test
+spec:
+  containers:
+ - name: container-test
+    image: busybox
+    volumeMounts:
+    - name: all-in-one
+      mountPath: "/projected-volume"
+      readOnly: true
+  volumes:
+  - name: all-in-one
+    projected:
+      sources:
+      - secret:
+          name: mysecret
+          items:
+            - key: username
+              path: my-group/my-username
+      - secret:
+          name: mysecret2
+          items:
+            - key: password
+              path: my-group/my-password
+              mode: 511
+```
+
+Each volume source is listed in the spec under sources. As stated above the
```

Comment:
@jpeeler You should use the code format for fields you are talking about that are in the yaml. You can use the tickmark markdown to do this, e.g. \`secretName\` => `secretName`. mburke5678's feedback has several good examples of its usage.